plugins/jobs/src/js/__tests__/JobsRunNow-test.js

```diff
@@ -0,0 +1,23 @@
+import { Observable } from "apollo-link";
+
+jest.mock("data-service");
+const jobsRunNow = require("../JobsRunNow");
+
+describe("Jobs Run Now Action", function() {
+  it("runNowJobGraphql call if made with onItemSelect", function(done) {
+    const runNowJobGraphql = jest.fn().mockReturnValue(
+      new Observable(() => {
+        expect(runNowJobGraphql).toBeCalled();
+        done();
+      })
+    );
+
+    const jobsRunNowAction = jobsRunNow.jobsRunNowAction(
```

Comment:
Do I get that right that we have no test which verifies that the GraphQL schema goes together with what we expect the client to deliver?

Reply:
Yeah, no test for that. Would we want a test for this?